llama-index-networks/llama_index/networks/network/retriever.py

```diff
@@ -15,17 +16,36 @@ class NetworkRetriever(BaseRetriever):
     def __init__(
         self,
         contributors: List[ContributorRetrieverClient],
+        reranker: Optional[BaseNodePostprocessor] = None,
```

Comment:
Couldn't this be any node post processor? Not just a reranker?

Reply:
I guess it could be yeah. I'll just rename this to `node_postprocessor`

Reply:
aha, found [this](https://github.com/run-llama/llama_index/blob/ef54ac6b7dcf9b91b2278a66d5699f1aa1eb70c1/llama-index-core/llama_index/core/chat_engine/condense_plus_context.py#L169) in our code base - will adopt this pattern as well.